doc/source/examples/analysis/polymers_and_membranes/hole2.ipynb

```diff
@@ -33,7 +33,7 @@
   },
```

Comment:
<p>you'll need to add installing hole to CI, see: https://github.com/MDAnalysis/mdanalysis/blob/89d19d55349772be44a3e08eab8e54f586a3fd97/.github/actions/build-src/action.yaml#L39-L50</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>